muxers/mplex/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "libp2p-mplex"
 edition = "2018"
 description = "Mplex multiplexing protocol for libp2p"
-version = "0.30.0"
+version = "0.31.0"
```

```suggestion
version = "0.30.1"
```

Is this a breaking change? If not, why not release it as a patch release?